apps/staking/src/api.ts

```diff
@@ -430,20 +430,24 @@ export const unstakeIntegrityStaking = async (
 };
 
 export const reassignPublisherAccount = async (
-  _client: PythStakingClient,
-  _stakeAccount: PublicKey,
-  _targetAccount: PublicKey,
+  client: PythStakingClient,
+  stakeAccount: PublicKey,
+  targetAccount: PublicKey,
+  publisherKey: PublicKey,
```

Comment:
The reason I did that was to avoid fetching this data. I can fetch this data from the sdk by making an rpc call but I wanted to avoid increasing the number of rpc call. Do you still think it's better to do it this way?

Reply:
hm, I'm not sure if I really have a strong opinion, let's leave it as is and revisit later.